src/main/java/de/retest/recheck/configuration/ProjectConfiguration.java

```diff
@@ -0,0 +1,88 @@
+package de.retest.recheck.configuration;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.stream.Stream;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ProjectConfiguration {
+
+	private static final Logger logger = LoggerFactory.getLogger( ProjectConfiguration.class );
+	private static final String RETEST_PROJECT_DEFAULTS = "retest-project-defaults.properties";
+	private static ProjectConfiguration INSTANCE;
+
+	public static final String RETEST_PROJECT_PROPERTIES = "retest.properties";
```

Comment:
Isn't there already a constant for this?

Reply:
I would like this configuration to be clearly separated from all old stuff.